### PR TITLE
Add RhinoSecurityLabs AWS IAM Privilege Escalation tool

### PR DESCRIPTION
**Pull request by cyberqueenmeg**
Hi,

I've revamped the RhinoSecLabs AWS IAM privesc tool (it was old), and would love to add it to blackarch. Thanks!

### CODE REVIEW

File: packages/aws_iam_privesc/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=aws_iam_privesc
+pkgver=33.f5c92f9
+pkgrel=2
+pkgdesc='AWS IAM policy scanner that helps determine where privilege escalation can be achieved'
+arch=('any')
+groups=('blackarch' 'blackarch-scanner' 'blackarch-recon' 'blackarch-exploitation' 'blackarch-automation')
+url='https://github.com/cyberqueen-meg/log4j-bypass'
```
Comment (noraj) [inline packages/aws_iam_privesc/PKGBUILD:10]:
```suggestion
url='https://github.com/cyberqueenmeg/aws_iam_privesc'
```

File: packages/aws_iam_privesc/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+license=('MIT')
+depends=('python' 'python-requests' 'python-pycryptodome' 'python-termcolor' 'python-boto3' 'python-botocore' 'aws' )
+makedepends=('git')
+source=("git+https://github.com/cyberqueen-meg/$pkgname.git")
```
Comment (noraj) [inline packages/aws_iam_privesc/PKGBUILD:14]:
```suggestion
source=("git+https://github.com/cyberqueenmeg/$pkgname.git")
```

File: packages/aws_iam_privesc/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,42 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=aws_iam_privesc
```
Comment (noraj) [inline packages/aws_iam_privesc/PKGBUILD:4]: Maybe

```suggestion
pkgname=aws-iam-privesc
_pkgname=aws_iam_privesc
```

and replace `$pkgname` with `$_pkgname`

File: packages/aws_iam_privesc/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  install -Dm 644 -t "$pkgdir/usr/share/doc/$pkgname/" README.md
+  install -Dm 644 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
+
+  rm -rf LICENSE README.d
+
+  cp -a * "$pkgdir/usr/share/$pkgname/"
+
+  cat > "$pkgdir/usr/bin/$pkgname" << EOF
+#!/bin/sh
+cd /usr/share/$pkgname
```
Comment (noraj) [inline packages/aws_iam_privesc/PKGBUILD:38]: If the following works it's better not to `cd`

```
exec python3 /usr/share/$pkgname/aws_escalate.py "\$@"
```

**Review by noraj — CHANGES_REQUESTED** [review]

File: packages/aws_iam_privesc/PKGBUILD (hunk 4 above)
Comment (cyberqueenmeg) [reply packages/aws_iam_privesc/PKGBUILD]: updated the code

**Comment by cyberqueenmeg** [comment]
@noraj - thanks for the feedback, I made all requested changes.

**Review by noraj — APPROVED** [review]
LGTM

**Comment by noptrix** [comment]
@cyberqueenmeg thanks for this. please check my changes for future usage of PKGBUILD of such a package.

**Pull request merged**